### PR TITLE
Skip drawing anything if VECTOR_GRAPHICS_SKIP_RASTER is true in debug mode

### PR DESCRIPTION
**Pull request by jonahwilliams**

### CODE REVIEW

File: packages/vector_graphics/lib/src/render_vector_graphics.dart [hunk 1]
```diff
@@ -2,6 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:io';
```
Comment (dnfield) [inline packages/vector_graphics/lib/src/render_vector_graphics.dart:5]: This breaks web support right?

File: packages/vector_graphics/lib/src/render_vector_graphics.dart [hunk 2]
```diff
@@ -130,6 +131,13 @@ class RenderVectorGraphic extends RenderBox {
 
   @override
   void paint(PaintingContext context, ui.Offset offset) {
+    if (kDebugMode) {
+      final String? skip = Platform.environment['VECTOR_GRAPHICS_SKIP_RASTER'];
+      if (skip == 'true') {
+        return;
```
Comment (dnfield) [inline packages/vector_graphics/lib/src/render_vector_graphics.dart:137]: Draw a rect with some weird color?

File: packages/vector_graphics/lib/src/render_vector_graphics.dart (hunk 1 above)
Comment (jonahwilliams) [reply packages/vector_graphics/lib/src/render_vector_graphics.dart:5]: yeah - I could conditionally import it, we don't have any web users yet though

File: packages/vector_graphics/lib/src/render_vector_graphics.dart (hunk 2 above)
Comment (jonahwilliams) [reply packages/vector_graphics/lib/src/render_vector_graphics.dart:137]: what is your favorite?

Comment (jonahwilliams) [reply packages/vector_graphics/lib/src/render_vector_graphics.dart]: picked fuchsia

File: packages/vector_graphics/lib/src/render_vector_graphics.dart (hunk 1 above)
Comment (jonahwilliams) [reply packages/vector_graphics/lib/src/render_vector_graphics.dart]: done

File: packages/vector_graphics/lib/src/_debug_web.dart [hunk 3]
```diff
@@ -0,0 +1,8 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Skip rasterization on web platform debug mode.
+bool get debugSkipRaster {
+  return true;
```
Comment (dnfield) [inline packages/vector_graphics/lib/src/_debug_web.dart:7]: This seems likea bad default until we have web test users. 

Maybe use the zone thing for now?

Comment (jonahwilliams) [reply packages/vector_graphics/lib/src/_debug_web.dart:7]: I'd rather leave it on then use Zones for anything

**Review by dnfield — APPROVED** [review]

**Pull request merged**